lib/mixins/id.js

```diff
@@ -24,5 +24,11 @@ export default {
         safeId() {
             return this.id || this.localId_ || null;
         }
+    },
+    methods: {
+        safeIdPlus(str) {
+            str = String(str || '').replace(/\s+/g, '_');
+            return (this.safeId && Boolean(str)) ? `${this.safeId}_${str}` : null;
+        }
     }
 };
```

Comment:
but I could convert `safeId` to a method, and optionally append the first argument